<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FmMain</class>
 <widget class="QMainWindow" name="FmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TTF: what changed?</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="wiIn" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2">
        <widget class="QLineEdit" name="edNew"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lbOld">
         <property name="text">
          <string>Old TTF</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbNew">
         <property name="text">
          <string>New TTF</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="edOld"/>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btBrowseNew">
         <property name="text">
          <string>Browse…</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btBrowseOld">
         <property name="text">
          <string>Browse…</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btGo">
           <property name="text">
            <string>Go!!</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="memoResult"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edOld</tabstop>
  <tabstop>btBrowseOld</tabstop>
  <tabstop>edNew</tabstop>
  <tabstop>btBrowseNew</tabstop>
  <tabstop>btGo</tabstop>
  <tabstop>memoResult</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
